File: Frontend/src/app/model/Report.ts
export class Report {
    reportId: number;
    type: string;
    dateGenerated: string;
    status: string;
    description: string;
    doctorId: number;
    patientId: number;
  
    constructor(
      reportId: number,
      type: string,
      dateGenerated: string,
      status: string,
      description: string,
      doctorId: number,
      patientId: number
    ) {
      this.reportId = reportId;
      this.type = type;
      this.dateGenerated = dateGenerated;
      this.status = status;
      this.description = description;
      this.doctorId = doctorId;
      this.patientId = patientId;
    }
}

File: Backend/src/main/java/edu/icet/dto/Report.java
package edu.icet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Report {
    private int reportId;
    private String type;
    private String dateGenerated;
    private String status;
    private String description;
    private int doctorId;
    private int patientId;
}
